proposals/field-keyword.md

```diff
@@ -598,3 +598,44 @@ partial class C
 ```
 
 **Recommendation**: Disallow auto-accessors in partial property implementations, because the limitations around when they would be usable are more confusing to follow than the benefit of allowing them.
+
+### Readonly field
+
+When should the synthesized backing field be considered *read-only*?
+
+```csharp
+struct S
+{
+    object P1          { get { field ??= ""; }}        // ok
+    readonly object P2 { get { field ??= ""; }}        // error: 'field' is readonly
+    readonly object P3 { get; set { _ = field; } }     // ok
+    readonly object P4 { get; set { field = value; } } // error: 'field' is readonly
```

Comment:
can you add a line where the property has an initializer?

Reply:
The first line (P0) has one, is that different from what you'd want to see?

Reply:
I added the `P0` example based on the comment from @CyrusNajmabadi.

Reply:
Ah, thanks! Sorry Cyrus!